netsim/modules/vlan.yml

```diff
@@ -55,13 +55,17 @@ attributes:
     bandwidth:
     _selfloop_ifindex:
     stp:
+    virtual_interface:  # Use case: VLAN on lag interface
+    lag:                # Keep lag parameters such as lacp settings
+  
   #
   # Keep these subinterface attributes
   keep_subif:
     vlan:
     ifindex:
     ifname:
     type:
+    virtual_interface:
```

Comment:
This seems wrong. There is no need to keep the "virtual_interface" setting on a subinterface (which is a virtual interface anyway).

Reply:
removed